summary: age buckets added (#193)
tarantool/metrics: README.md

```lua
local metrics = require('metrics')

-- create a summary with a sliding wingow with 4 age buckets and 600s bucket lifetime
local http_requests_latency = metrics.summary(
    'http_requests_latency', 'HTTP requests total',

>     {[0.5]=0.01, [0.9]=0.01, [0.99]=0.01, 600, 4}

Reviewer: README contradicts api_reference. api_reference points out that it should be 
```
local http_requests_latency = metrics.summary(
    'http_requests_latency', 'HTTP requests total',
    {[0.5]=0.01, [0.9]=0.01, [0.99]=0.01}
    {[0.5]=0.01, [0.9]=0.01, [0.99]=0.01},
    600,
    4
)
```

Reviewer: It is still relevant

```lua
local metrics = require('metrics')

> -- create a summary with a sliding wingow with 5 age buckets and 60s bucket lifetime

Reviewer 2: ```suggestion
-- create a summary with a sliding window of 5 age buckets and 60s bucket lifetime
```

> -- create a summary with a sliding wingow with 5 age buckets and 600s bucket lifetime

Reviewer: To be honest, it's pretty hard to understand how exactly sliding window works without explicit explanation in PR description

   local metrics = cartridge.service_get('metrics')
   ```

5. There is an ability in Tarantool Cartridge >= '2.4.0' to set a zone for each

>    server in cluster. If zone was set for the server 'zone' label for all metrics

Reviewer 2: ```suggestion
   server in a cluster. If a zone was set for the server 'zone' label will be added
```

>    of this server will be added.

Reviewer 2: ```suggestion
   for all metrics on this server.
```
